quickstarts/microsoft.maps/maps-use-sas/azuredeploy.json

```diff
@@ -0,0 +1,185 @@
+            }
+        },
+        "startDateTime": {
+            "type": "string",
+            "defaultValue": "[utcNow('u')]",
+            "metadata": {
+                "description": "Current Universal DateTime in ISO 8601 'u' format to be used as start of the SAS Token."
+            }
+        },
+        "duration" : {
+            "type": "string",
+            "defaultValue": "P1Y",
+            "metadata": {
+                "description": "The duration of the SAS Token, P1Y is maximum, ISO 8601 format is expected."
+            }
+        },
+        "maxRatePerSecond": {
+            "type": "int",
+            "defaultValue": 500,
+            "minValue": 1,
+            "maxValue": 500,
+            "metadata": {
+                "description": "The approximate maximum rate per second the SAS token can be used."
+            }
+        },
+        "signingKey": {
+            "type": "string",
+            "defaultValue": "primaryKey",
+            "allowedValues": [
+                "primaryKey",
+                "seconaryKey"
+            ],
+            "metadata": {
+                "description": "The specified signing key which will be used to create the SAS Token."
+            }
+        },
+        "allowedOrigins": {
+            "type": "array",
+            "defaultValue": [],
+            "maxLength": 10,
+            "metadata": {
+                "description": "The specified application's web host header origins (example: https://www.azure.com) which the Maps account allows for Cross Origin Resource Sharing (CORS)."
+            }
+        }
+    },
+    "variables": {
+        "accountId": "[resourceId('Microsoft.Maps/accounts', parameters('accountName'))]",
+        "Azure Maps Data Reader": "[subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '423170ca-a8f6-4b0f-8487-9e4eb8f49bfa')]",
+        "sasParameters": {
+            "signingKey": "[parameters('signingKey')]",
+            "principalId": "[parameters('userAssignedIdentityPrincipalId')]",
+            "maxRatePerSecond": "[parameters('maxRatePerSecond')]",
+            "start": "[parameters('startDateTime')]",
+            "expiry": "[dateTimeAdd(parameters('startDateTime'), parameters('duration'))]"
+        }
+    },
+    "resources": [
+        {
+            "name": "[parameters('accountName')]",
+            "type": "Microsoft.Maps/accounts",
+            "apiVersion": "2021-12-01-preview",
+            "location": "[parameters('location')]",
+            "sku": {
+                "name": "[parameters('pricingTier')]"
+            },
+            "kind": "[parameters('kind')]",
+            "properties": {
+                "cors": {
+                    "corsRules": [
+                        {
+                            "allowedOrigins": "[parameters('allowedOrigins')]"
+                        }
+                    ]
+                }
+            },
+            "identity": {
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties